src/routes/(authenticated)/profile/[user_id]/edit-profile/change-email/+page.svelte

```diff
@@ -0,0 +1,72 @@
+<script lang="ts">
+	import type { PageData } from './$types';
+	import { superForm } from 'sveltekit-superforms';
+
+	export let data: PageData;
+
+	const { enhance, form, errors, constraints, message } = superForm(data.form);
+
+    console.log('In the front : ', form)
+    console.log('In the front : ', errors)
```

Comment:
Have a look at:
https://superforms.rocks/super-debug
This helps a lot if you're struggling to debug the superforms